README.adoc

```diff
@@ -28,56 +31,25 @@ Please ensure that you make PRs and issues in the appropriate repository!
 
 ====
 
-[[cache-warning]]
```

Comment:
I guess this is no longer such a big deal now we have the config in flake.nix. Hooray!

Reply:
🥳  the cache warning has been rephrased and moved to the `How to get a shell environment with tools` section